### DISCUSSION TITLE
Is there something like "os" python library for Z#

### DISCUSSION THREAD

**Discussion by Rexo35** [sam-astro/Z-Sharp#42] [Q&A]
I want to make an screen cleaning in a console aplication. Is there any way to do "cls" in Z#? In python it would look like this:
`from os import system as sys`
`sys("cls")`

**Comment by sam-astro** [comment]
No, the only console functions are `Print` and `Printl`

**Reply by Rexo35** [reply]
Now there is an "ZS.System.Command" function, just if someone is interested.
